docs/changes/coordinates/17263.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix a crash when instantiating ``Angle`` from a non-numpy array (for instance
+pyarrow arrays).
```

Comment:
If this bug also affected `Longitude` and `Latitude` then the change log entry should mention them too.